app/assets/stylesheets/components/page_specific/_home_top.scss

```diff
@@ -0,0 +1,184 @@
+// Banner at the top of the home page
+//
+// Styleguide Home Top
+
+.home-top {
+  background: image_url('illustrated_people.png') 0 100% no-repeat;
+  background-size: 135%;
+  position: relative;
+  padding-bottom: 50%;
+
+  @include respond-to($mq-m) {
+    background-position: 165% 108%;
+    background-size: 70%;
+    padding-bottom: 0;
+  }
+
+  @include respond-to($mq-l) {
+    background-size: 65%;
+    background-position: 141% 100%;
+  }
+
+  @include respond-to($mq-xl) {
+    background: none;
+    overflow: hidden;
+  }
+}
+
+.home-top-trust {
+  position: relative;
+
+  @include respond-to($mq-xl) {
+    position: absolute;
+    top: 0;
+    width: 1000px;
+  }
+}
+
+.home-top__image {
+  @include respond-to($mq-xl) {
+    width: 1350px;
+    height: 309px;
+    margin: 0 auto;
+    position: relative;
+    background: image_url('illustrated_people.png') 100% 52% no-repeat;
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings